pagerduty/resource_pagerduty_team.go

```diff
@@ -56,7 +60,7 @@ func resourcePagerDutyTeamCreate(d *schema.ResourceData, meta interface{}) error
 
 	d.SetId(team.ID)
 
-	return nil
+	return resourcePagerDutyTeamRead(d, meta)
```

Comment:
Without this, the following error occurs:

```
# make testacc TEST=./pagerduty TESTARGS='-run=TestAccPagerDutyTeam_Basic -count=1'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./pagerduty -v -run=TestAccPagerDutyTeam_Basic -count=1 -timeout 120m
=== RUN   TestAccPagerDutyTeam_Basic
--- FAIL: TestAccPagerDutyTeam_Basic (5.14s)
    testing.go:568: Step 0 error: Check failed: Check 4/4 error: pagerduty_team.foo: Attribute 'html_url' expected to be set
FAIL
FAIL    github.com/terraform-providers/terraform-provider-pagerduty/pagerduty   5.156s
FAIL
make: *** [GNUmakefile:17: testacc] Error 1
```

See https://github.com/terraform-providers/terraform-provider-pagerduty/pull/162/files#r336729232